ts/store/reducers/index.ts

```diff
@@ -22,8 +22,22 @@ import notificationsReducer from "./notifications";
 import onboardingReducer from "./onboarding";
 import pinloginReducer from "./pinlogin";
 import profileReducer from "./profile";
+import { GlobalState } from "./types";
 import walletReducer from "./wallet";
 
+/** State keys we want to retail on user logout.
```

Comment:
retail -> retain

Reply:
why we want to keep these keys? a logout should completely reset the application state